src/elements/content-uploader/OverallUploadsProgressBar.js

```diff
@@ -73,6 +75,10 @@ const OverallUploadsProgressBar = ({ percent, view, onClick, onKeyDown, isDraggi
     return (
         <div
             className="bcu-overall-progress-bar"
+            data-resin-target={classNames({
```

Comment:
Should we use `classNames` for non-CSS purposes? I had to look at this a few times to realize it wasn't setting actual class names.